doc/sphinx/source/ops/broadcastdistributed.rst

```diff
@@ -0,0 +1,46 @@
+.. broadcastdistributed.rst:
+
+#########
+BroadcastDistributed
+#########
```

Comment:
Heading designators need to be the length of the string you're wanting to be the heading.  So here we need:  

```
#################
BroadcastDistributed
#################```